src/schema/questionnaire.ts

```diff
@@ -0,0 +1,25 @@
+import { Type } from "@sinclair/typebox";
+/**
+ * Schema for the Questionnaire table
+ */
+
+const questionnaireSchema = Type.Object({
+  title: Type.String(),
+  description: Type.String(),
+  options: Type.Array(
+    Type.Object({
+      question: Type.String(),
+      options: Type.Array(
+        Type.Object({
+          label: Type.String(),
+          value: Type.Boolean(),
+        }),
+      ),
+    }),
+  ),
+  tags: Type.Optional(Type.Array(Type.String())),
+  passedUsers: Type.Optional(Type.Array(Type.Number())),
```

Comment:
This schema seems fine, however I notice only tags and passed users are optional (also fine) but in the home-lambdas-api swagger the questionnaire schema does not have any required properties. This could cause issue down the line so I would suggest creating a task in trello for updating the spec to contain the appropriate required values. This feature can be merged (and even tested) before the spec is updated but I would not recommend doing any UI work using the lambdas until the spec is updated.

Reply:
Made new Trellocard for this. Added parts of this comment to that.